src/component/sheet.js

```diff
@@ -721,6 +723,10 @@ function sheetInitEvents() {
 
   bind(window, 'click', (evt) => {
     this.focusing = overlayerEl.contains(evt.target);
+    if (!overlayerEl.contains(evt.target)) {
+      this.focusing = false;
+      this.trigger('blur');
+    }
```

Comment:
```suggestion
    if (!this.focusing) this.trigger('blur');
```
this.focusing should already be false by the line above right?